addons/mrp/static/src/views/bom_line_list.js

```diff
@@ -0,0 +1,27 @@
+import { useService } from "@web/core/utils/hooks";
+import { registry } from "@web/core/registry";
+import { ListController } from "@web/views/list/list_controller";
+import { listView } from "@web/views/list/list_view";
+
+export class BomLineListController extends ListController {
+    setup() {
+        super.setup();
+        this.action = useService("action");
+    }
+
```

Comment:
`ListController` already uses this service (see `this.actionService`)
```suggestion
```

Reply:
Removed, thanks! :+1: